pubsub/pullstream.go

```diff
@@ -47,12 +48,14 @@ func newPullStream(ctx context.Context, streamingPull streamingPullFunc, subName
 			spc, err := streamingPull(ctx, gax.WithGRPCOptions(grpc.MaxCallRecvMsgSize(maxSendRecvBytes)))
 			if err == nil {
 				recordStat(ctx, StreamRequestCount, 1)
+				log.Print("starting open")
 				err = spc.Send(&pb.StreamingPullRequest{
 					Subscription: subName,
 					// We modack messages when we receive them, so this value doesn't matter too much.
 					StreamAckDeadlineSeconds: 60,
 				})
 			}
+			log.Printf("open returned %v", err)
```

Comment:
This isn't immediately after the Send call. It is after the if statement. Move this up one line.